### DISCUSSION TITLE
Custom component not found in integration folder

### DISCUSSION THREAD

**Discussion by TheUnseenBug** [faanskit/ha-checkwatt#76] [General]
Hello,
I am using a homeassistant green and have git cloned the repository under the config folder in a custom_components folder and I can see all the data using the terminal however when going to the add integration popup Checkwatt is not found. I have tried restarting the homeassistant and hard refreshing and using a new browser so should not be a cookies issue atleast.

**Comment by faanskit** [comment]
Please give some more details, impossible to support with this little information.
Can you please give the outline of the directory `custom_components`, i.e. like this:

```
# ~ cd ~/config/custom_components
# pwd                          
/root/config/custom_components
# ls -la                       
total 32
drwxr-xr-x    8 root     root          4096 Jun 23 08:25 .
drwxr-xr-x   10 root     root          4096 Jul 11 11:33 ..
drwxr-xr-x    4 root     root          4096 Jun 23 08:25 checkwatt
drwxr-xr-x    4 root     root          4096 May 20 12:45 easee
drwxr-xr-x   10 root     root          4096 Mar 28 11:44 hacs
drwxr-xr-x    4 root     root          4096 Mar 28 14:12 nordpool
drwxr-xr-x    4 root     root          4096 May 20 12:35 openai_tts
drwxr-xr-x    4 root     root          4096 Mar 28 11:48 plejd
```

**Comment by TheUnseenBug** [comment]
![bild](https://github.com/user-attachments/assets/457d6683-7d59-4e2f-b559-677668c70296)
This is from the config folder and downwords.

**Reply by faanskit** [reply]
You have not installed it correct.  This will, and should not, work.
I recommend to remove all that and start over, and if possible use Hacs.

If not, git clone into a temp directory and move folder checkwatt to custom_components.

The below may fix it for you:

```
# ~ cd ~/config/custom_components
# mv checkwatt tmp_cw
# mv tmp_cw/custom_components/checkwatt checkwatt
# rm -rf tmp_cw

```
